### ISSUE TITLE
Invalid code generated with prettier-ignore inside of object

### ISSUE THREAD

**Issue by vjeux** [prettier/prettier#3447]
**Prettier 1.9.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAnOaAO6cmAvJsADpSaYD0tm+uMMAlnNgLSsDmUEuKjQDkAfXgAPGMKSZhAFQCu2atEwwAFsQA2vDTDTCANEMytYCGLOG6e+wyagBfANxUQRkBHxt0yUACG2NgQAO4ACkEIaMggAQBuEKwAJh5xGMgAZgHaaHCeAEbYAWAA1nAwAMr4JeY8yDDYivkgyRBgWTl5nuZ52DDhxTwAtgGduS0AVmgSAELFZRWVAcNwADLmcOPdXoow+HsATNstNdh9sQUBBQCe2tBpTOYwAOopmsgAHAAMnkwQeRexXwsSYeA48S2nlwAEdFKxcIMAiMxkhshNPHlhqwGk0Wmg6to4ABFRQQeAnTwwa5vZIfJCHKnFVi6KA8ADCEGGo1i-CgUJAijy8muMTRXTgTicQA)

**Input:**
```jsx
const response = {
  // prettier-ignore
  '_text': 'Turn on the lights',
  intent: 'lights',
};

```

**Output:**
```jsx
const response = {
  // prettier-ignore
  '_text': 'Turn on the lights'
  intent: "lights"
};

```

**Expected behavior:**
The `,` should not be removed!

**Comment by duailibe** [comment]
It's a regression from #3103

**Comment by vjeux** [comment]
This causes issues on 4 files inside of Facebook for context. This is the only issue that is found by --debug-check on the biggest fb repo :)

**Comment by duailibe** [comment]
Fixed on #3448
